src/main/webapp/app/exercises/text/participate/text-editor.component.ts

```
@@ -176,6 +252,11 @@
                 });
             }
         }
+
+        this.participationUpdateListener?.unsubscribe();
+        if (this.participation) {
+            this.participationWebsocketService.unsubscribeForLatestResultOfParticipation(this.participation.id!, this.textExercise);
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)